secure-mobile-push/smp-android-client/app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,6 @@
 <resources>
     <string name="app_name">Secure Mobile Push</string>
+    <string name="send_a_sms_for_app_to_read">Send a SMS for the App to Read</string>
```

Comment:
please rename to 

- text_sms_send
- text_sms_sender_number
- text_sms_message

and change string to "Send a SMS to the application"